packages/flutter_form_bloc/pubspec.yaml

```diff
@@ -4,16 +4,22 @@ version: 0.31.0
 homepage: https://github.com/GiancarloCode/form_bloc/tree/master/packages/flutter_form_bloc
 repository: https://github.com/GiancarloCode/form_bloc/tree/master/packages/flutter_form_bloc
 issue_tracker: https://github.com/GiancarloCode/form_bloc/issues
+publish_to: none
 
 environment:
-  sdk: '>=2.12.0 <4.0.0'
+  sdk: ">=2.12.0 <4.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
   flutter_bloc: ^8.0.1
-  form_bloc: ^0.31.0
+  # form_bloc: ^0.31.0
+  form_bloc:
+    git:
+      url: https://github.com/Cavin6080/form_bloc.git
+      path: packages/form_bloc
+
```

Comment:
I think a PR should not change this to point to another repository. This should keep pointing to the same repo. Use paths if needed.

Reply:
I had to do this because if we want to use flutter_form_bloc directly from GIT then we can't use it if we mention paths in `pubspec.yaml`.  I've made the changes now for this and pushed this.